README.md

```diff
@@ -64,7 +74,7 @@ We also have official client-side implementations for
 [Java](https://github.com/metarhia/jstp-java)
 that work effortlessly on iOS and Android 🎉
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```

Comment:
I like the badges :smile:. Maybe we will move three badges under `Installation` up below the first 5 badges (on a new line, centered) and make all titles beside `JSTP` as they were before?